proxy/src/registry/transaction.rs

```diff
@@ -109,6 +109,14 @@ pub enum Message {
         /// Identity id originated from librad.
         id: Option<String>,
     },
+
+    /// Issue a member registration for a given handle under a given org.
+    MemberRegistration {
```

Comment:
We should force consistent camelcasing by using `#[serde(rename_all = "camelCase")]` on the variants as well.

Reply:
Does [L83](https://github.com/radicle-dev/radicle-upstream/blob/0f5dd37f97d869dc5f5a7cff2d75df8197d2b953/proxy/src/registry/transaction.rs#L83) not affect the variants as well?

Reply:
No unfortunately it doesn't as serde treats every struct variant as its own container.

Reply:
addressed in #449